bluebottle/members/admin.py

```diff
@@ -111,18 +112,38 @@ class MemberVotesInline(admin.TabularInline):
 
 
 class MemberAdmin(UserAdmin):
-    standard_fieldsets = (
-        (None, {'fields': ('email', 'password', 'remote_id')}),
-        (_('Personal info'), {'fields': (
-            'first_name', 'last_name', 'username', 'gender', 'birthdate',
-            'phone_number')}),
-        (_("Profile"),
-         {'fields': ('user_type', 'is_co_financer', 'can_pledge', 'picture', 'about_me', 'location')}),
-        (_("Settings"), {'fields': ['primary_language', 'newsletter']}),
-        (_('Skills & interests'), {'fields': ('favourite_themes',)}),
-        (_('Important dates'),
-         {'fields': ('last_login', 'date_joined', 'deleted')}),
-    )
+    def get_standard_fieldsets(self):
+
+        standard_fieldsets = (
+            (None, {'fields': ('email', 'password', 'remote_id')}),
+            (_('Personal info'), {'fields': (
+                'first_name', 'last_name', 'username', 'gender', 'birthdate',
+                'phone_number')}),
+            (_("Profile"),
+             {'fields': ('user_type', 'is_co_financer', 'picture', 'about_me', 'location')}),
+            (_("Settings"), {'fields': ['primary_language', 'newsletter']}),
+            (_('Skills & interests'), {'fields': ('favourite_themes',)}),
+            (_('Important dates'),
+             {'fields': ('last_login', 'date_joined', 'deleted')}),
+        )
+
+        for item in properties.PAYMENT_METHODS:
+            if item['name'] == 'Pledge':
+                standard_fieldsets = (
+                    (None, {'fields': ('email', 'password', 'remote_id')}),
+                    (_('Personal info'), {'fields': (
+                        'first_name', 'last_name', 'username', 'gender', 'birthdate',
+                        'phone_number')}),
+                    (_("Profile"),
+                     {'fields': ('user_type', 'is_co_financer', 'can_pledge', 'picture', 'about_me', 'location')}),
+                    (_("Settings"), {'fields': ['primary_language', 'newsletter']}),
+                    (_('Skills & interests'), {'fields': ('favourite_themes',)}),
+                    (_('Important dates'),
+                     {'fields': ('last_login', 'date_joined', 'deleted')}),
+                )
+                break
```

Comment:
Why don't you add the field to the configuration instead of copying everything over? 

I would also just remove the break. Looping over 2 more payment methods will not kill performance.